### PR TITLE
updates is-svg to 4.3.0; associated dependency updates

### PR DESCRIPTION
**Pull request by shuldt**

### CODE REVIEW

**Comment by erbridge** [comment]
You might want to look at this instead of putting things in the dependencies list: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/

**Comment by shuldt** [comment]
> You might want to look at this instead of putting things in the dependencies list: https://classic.yarnpkg.com/lang/en/docs/selective-version-resolutions/

yup this looks very sensible, i'll try it thank you

File: package.json [hunk 1]
```diff
@@ -8,8 +8,10 @@
     "@rails/webpacker": "^5.4.3",
     "glob-parent": "^6.0.2",
     "govuk-frontend": "^4.0.0",
+    "is-svg": "^4.3.0",
     "jquery": "^3.6.0",
-    "normalize-url": "7.0.2"
+    "normalize-url": "7.0.2",
+    "postcss-svgo": "4.0.3"
```
Comment (erbridge) [inline package.json:14]: I'd suggest we don't want the dependencies added. They're not direct dependencies and adding them here makes them look like they are. That's what the resolutions are for - to resolve indirect dependencies.

**Review by erbridge — CHANGES_REQUESTED** [review]
I'd be happy to chat quickly about what I think needs to change if helpful.

File: Gemfile.lock [hunk 2]
```diff
@@ -307,6 +307,8 @@ PLATFORMS
   x86_64-darwin-19
   x86_64-darwin-20
   x86_64-linux
+  x86_64-darwin-21
+  x86_64-linux
```
Comment (erbridge) [inline Gemfile.lock:311]: Are supposed to have linux twice?

**Review by erbridge — APPROVED** [review]

**Pull request merged**